packages/design/tailwind/css/components.css

```diff
@@ -944,6 +944,88 @@ input[type='file' i] {
 
 /* End of Breadcrumbs */
 
+/* Begin Progress Stepper */
+
+.gi-progress-stepper {
+  @apply gi-w-full gi-flex gi-items-start gi-overflow-y-auto;
+}
+
+.gi-progress-stepper[data-orientation='vertical'] {
+  @apply gi-flex-col;
+}
+
+.gi-progress-stepper-step {
+  @apply gi-text-white 
+  gi-rounded-full 
+  gi-relative
```

Comment:
not sure if this is a formatting issue but we have some classes that are being displayed line by line and some that are being displayed one after another in a single line, we should decide which formatting style we want to follow so we are all aligned

Reply:
good call, I believe it was something wrong with my lint plugin, going to put them in a line format.